src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/NteractKernelExtensionTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+using Assent;
+
+using FluentAssertions;
+
+using Microsoft.DotNet.Interactive.Formatting;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Interactive.ExtensionLab.Tests
+{
+    public class NteractKernelExtensionTests : IDisposable
+    {
+        private readonly Configuration _configuration;
+
+        public NteractKernelExtensionTests(ITestOutputHelper output)
+        {
+            _configuration = new Configuration()
+                .SetInteractive(Debugger.IsAttached)
+                .UsingExtension("json");
+        }
+
+        [Fact]
+        public async Task it_registers_formatters()
+        {
+            using var kernel = new CompositeKernel();
+
+            var kernelExtension = new NteractKernelExtension();
+
+            await kernelExtension.OnLoadAsync(kernel);
+
+            var data = new[]
+            {
+                new {Type="orange", Price=1.2},
+                new {Type="apple" , Price=1.3},
+                new {Type="grape" , Price=1.4}
+            };
+
+
+            var formatted = data.ToTabularJsonString().ToDisplayString(HtmlFormatter.MimeType);
+
+            formatted.Should().Match("*getExtensionRequire('nteract','1.0.0')(['nteract/index'], (nteract) => {*");
```

Comment:
`Should().Contain()` is probably better here.

Reply:
done